app/views/organise/index.html.erb

```diff
@@ -8,22 +8,47 @@
     Are you an meetup or conference organiser, speaker, speaker in the making or someone that feels this event is important and has to happen in your city?
   </p>
   <p>
-    In that case our aim is to make it as simple as possible for you to organise a workshop for March 2nd. Sign up and add the details of your workshop, invite your facilitator and mentors too and your workshop details will be added to the site after being reviewed.
+    In that case our aim is to make it as simple as possible for you to organise a workshop as part of Global Diversity CFP Day. Sign up and add the details of your workshop, invite your facilitator and mentors too and all of this will be added to the site after being reviewed.
   </p>
   <p>
-    We'll give you a schedule and a set of educational videos to make sure you have as little to do as possible and can focus on providing attendees with a great experience.
+    We'll give you a schedule, a set of educational videos and activities to make sure you have as little to do as possible and can focus on providing attendees with a great experience.
   </p>
+  <p>
+    Is someone already running a workshop in your location?
+    <br/>
+    Reach out to them and offer to support their efforts. Many hands make light work.
+    <br/>
+    Global Diversity CFP Day simply would not happen without people coming together to make this event happen.
+  </p>
+
   <h3>
     What will you need to get this moving?
   </h3>
   <ol>
-    <li>Ensure you agree to our Code of Conduct as you will be in charge of enforcing it.</li>
-    <li>A warm postive approach to working with newbie speakers and place a great deal of value in the contribution they bring to this industry.</li>
-    <li>A venue with a projector and audio</li>
-    <li>A facilitator to introduce the sections of the workshop</li>
-    <li>A number of mentors, (around one mentor for every five attendees) who can advise attendees with their experience of applying to CFPs in the past.</li>
-    <li>A ticketing page</li>
-    <li>Be eager to promote your workshop to every tech community within 100 miles of the venue.</li>
+    <li>
+      A warm postive approach to working with newbie speakers from diverse backgrounds and place a great deal of value in the contribution they bring to this industry.
+    </li>
+    <li>
+      A venue with a projector and audio
+    </li>
+    <li>
+      An awesome team 🎉
+      <br/>
+      Your team will include a facilitator to introduce the sections of the workshop and mentors (around one mentor for every five attendees) who can advise attendees with their experience of applying to CFPs in the past.
+      <br/>
+      Representation Matters. When putting together your workshop team, the people involved need to reflect the people we are encouraging to attend.
```

Comment:
Representation Matters

So does transparency. 

Invite **all** team members to the site once you have a workshop setup, it'll encourage more potential attendees to participate.